test/success-tests/rebaseline.js

```diff
@@ -1,8 +1,9 @@
 "use strict";
-/* eslint-disable no-console */
 
-const path = require("path");
+const { EOL } = require("os");
```

Comment:
As before, no horrible Windows EOLs in my projects. Please revert all changes to this file.